docs/tutorial/work-pools.md

```diff
@@ -178,166 +140,145 @@ abc/workspace/123/deployments/deployment/xyz
 
 </div>
 
-Navigate to your Prefect Cloud UI and view your new deployment.
-Click the **Run** button to trigger a run of your deployment.
+## Schedule a deployment run
 
-Because this deployment was configured with a Prefect Managed work pool, Prefect Cloud will run your flow on your behalf.
+Now everything is set up for us to submit a flow-run to the work pool, using one of the following options:
 
-View the logs in the UI.
+* Use the CLI:
 
-### Schedule a deployment run
+    <div class="terminal">
 
-Now everything is set up for us to submit a flow-run to the work pool.
-Go ahead and run the deployment from the CLI or the UI.
+    ```bash
+    prefect deployment run 'get_repo_info/my-deployment'
+    ```
 
-<div class="terminal">
+    </div>
 
-```bash
-prefect deployment run 'get_repo_info/my-deployment'
-```
+* Use the UI, by navigating to your Prefect Cloud UI and viewing your new deployment. Click **Run** to trigger a run of your deployment that Prefect Cloud runs on your behalf.
 
-</div>
+    View the logs in the UI.
 
-Prefect Managed work pools are a great way to get started with Prefect.  
-See the [Managed Execution guide](/guides/managed-execution/) for more details.
+For more details on using Prefect-managed work pools, see [Managed Execution guide](/guides/managed-execution/).
 
-Many users will find that they need more control over the infrastructure that their flows run on.
-Prefect Cloud's push work pools are a popular option in those cases.
 
-## Push work pools with automatic infrastructure provisioning
+## Customize more granular infrastructure control
 
-Serverless push work pools scale infinitely and provide more configuration options than Prefect Managed work pools.
+For users requiring greater infrastructure control, Prefect Cloud offers push work pools. These serverless options:
```

Comment:
I would reserve a term like "control" for hybrid work pools with workers, and you have done at the end.